lib/ApplicationFeatures/V8PlatformFeature.cpp

```diff
@@ -81,10 +85,30 @@ static void gcEpilogueCallback(v8::Isolate* isolate, v8::GCType type,
   size_t usedHeapSize = h.used_heap_size();
   size_t stillFree = heapSizeLimit - usedHeapSize;
 
+  if (now - v8g->_lastMaxTime > 10*1000) {
```

Comment:
The `*1000` is not right here, as `now` and `_lastMaxTime` are measured in seconds, not milliseconds.